### PR TITLE
feat: peer support disable symmetric

### PR DESCRIPTION
**Pull request by qingwusunny**

### CODE REVIEW

**Review by KeyongSun — APPROVED** [review]

File: pkg/agent/controller/policy/policy_controller.go [hunk 1]
```diff
@@ -335,23 +336,23 @@ func (r *Reconciler) completePolicy(policy *securityv1alpha1.SecurityPolicy) ([]
...
+
 			if len(rule.From) == 0 {
+				ingressRule := ingressRuleTmpl.Clone()
 				// If "rule.From" is empty or missing, this rule matches all sources
 				ingressRule.SrcIPBlocks = map[string]*policycache.IPBlockItem{"": nil}
+				completeRules = append(completeRules, ingressRule)
 			} else {
-				// use policy namespace as ingress endpoint namespace
-				ingressRule.SrcGroups, ingressRule.SrcIPBlocks, err = r.getPeersGroupsAndIPBlocks(policy.Namespace, rule.From)
+				ingressRules, err := r.getCompleteRulesByParseSymmetricMode(ingressRuleTmpl, policy, networkingv1.PolicyTypeIngress, rule.From)
```
Comment (wu-cl) [inline pkg/agent/controller/policy/policy_controller.go:350]: networkingv1.PolicyTypeIngress is defined & used in NetworkPolicy. Although it has same str with ours, we'd better create a new const value?

File: plugin/tower/pkg/utils/testing/builder.go [hunk 2]
```diff
@@ -234,6 +234,19 @@ func RuleAddPorts(rule *v1alpha1.Rule, portInfo ...string) {
 	}
 }
 
+func RuleSetDisableSymmetric(rule *v1alpha1.Rule, disableSymmetric bool) {
+	if rule == nil {
+		return
+	}
+	if len(rule.From) > 0 {
+		rule.From[0].DisableSymmetric = disableSymmetric
```
Comment (wu-cl) [inline plugin/tower/pkg/utils/testing/builder.go:242]: Only first item used in Rule?

Comment (qingwusunny) [reply plugin/tower/pkg/utils/testing/builder.go:242]: plugin ut tool functions only support one rule.

File: pkg/agent/controller/policy/policy_controller.go (hunk 1 above)
Comment (qingwusunny) [reply pkg/agent/controller/policy/policy_controller.go:350]: securityPolicy use networkingv1.PolicyType defines it's policy type. So I use it.

**Review by wu-cl — APPROVED** [review]

**Pull request merged**